main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       alarmClock
+//  action 1:   the alarmClock ticks
+alarmClock.tick();
+//  action 2:   the alarmClock rings
+alarmClock.ring(); 
+//  action 3:   the alarmClock vibrates
+alarmClock.vibrate();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       scissor
+//  action 1:   the scissor cuts
+scissor.cut();
+//  action 2:   the scissor opens
+scissor.open();
+//  action 3:   the scissor glides
+scissor.glide();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       snow
+//  action 1:   the snow falls
+snow.fall();
+//  action 2:   the snow melts
+snow.melt();
+//  action 3:   the snow hardens
+snow.harden();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       bird
+//  action 1:   the bird flies
+bird.fly();
+//  action 2:   the bird sleeps
+bird.sleep();
+//  action 3:   the bird lays eggs
+bird.layEgg();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       milk
+//  action 1:   the milk spoils
+milk.spoil();
+//  action 2:   the milk spills
+milk.spill();
+//  action 3:   the milk churns
+milk.churn();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       tree
+//  action 1:   the tree falls
+tree.fall();
+//  action 2:   the tree shakes
+tree.shake();
+//  action 3:   the tree drops fruit
+tree.dropFruit();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       string
+//  action 1:   the string ties
+string.tie();
+//  action 2:   the string stretches
+string.stretch();
+//  action 3:   the string rips
+string.rip();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       tooth
+//  action 1:   the tooth aches
+tooth.ache();
+//  action 2:   the tooth chips
+tooth.chip();
+//  action 3:   the tooth falls out
+ tooth.fallOut();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.